lib/tasks/add_navigation_links.rake

```diff
@@ -1,25 +1,69 @@
 # rubocop:disable Metrics/BlockLength
 namespace :navigation_links do
+  reading_icon = File.read(Rails.root.join("app/assets/images/twemoji/drawer.svg")).freeze
+  contact_icon = File.read(Rails.root.join("app/assets/images/twemoji/contact.svg")).freeze
+  thumb_up_icon = File.read(Rails.root.join("app/assets/images/twemoji/thumb-up.svg")).freeze
+  smart_icon = File.read(Rails.root.join("app/assets/images/twemoji/smart.svg")).freeze
+  look_icon = File.read(Rails.root.join("app/assets/images/twemoji/look.svg")).freeze
+  listing_icon = File.read(Rails.root.join("app/assets/images/twemoji/listing.svg")).freeze
+  mic_icon = File.read(Rails.root.join("app/assets/images/twemoji/mic.svg")).freeze
+  camera_icon = File.read(Rails.root.join("app/assets/images/twemoji/camera.svg")).freeze
+  tag_icon = File.read(Rails.root.join("app/assets/images/twemoji/tag.svg")).freeze
+  bulb_icon = File.read(Rails.root.join("app/assets/images/twemoji/bulb.svg")).freeze
+  shopping_icon = File.read(Rails.root.join("app/assets/images/twemoji/shopping.svg")).freeze
+  heart_icon = File.read(Rails.root.join("app/assets/images/twemoji/heart.svg")).freeze
+  rainbowdev = File.read(Rails.root.join("app/assets/images/rainbowdev.svg")).freeze
+
+  desc "Create navigation links for new forem"
+  task create: :environment do
+    NavigationLink.where(url: "/readinglist").first_or_create(
+      name: "Reading List",
+      url: URL.url("readinglist"),
```

Comment:
In the `update` rake task (below mine), the variable `base_url` is built using `protocol` and `domain`.

However, I noticed that in the `app/lib/url.rb` file, the method `self.url` basically returns the same thing.

So I'm using `URL.url` in my rake tasks, and also in the specs. This choice avoids duplicating the `protocol` and `domain` variables. I had tried moving these two variables up like I did the link icons, but I kept getting an `UninitializedConstant: ApplicationConfig` error.

Now, the `update` rake task is called only in the `seeds.rb` file (which runs for dev environments only). I decided not to implement the use of `URL.url` in the `update` rake task, but I'm open to opinions.